gpib_server.py

```diff
@@ -112,7 +112,7 @@ def refreshDevices(self):
                     elif addr.startswith('TCPIP'):
                         instName = addr
                     elif addr.startswith('USB'):
-                        instName = addr + '::INSTR'
+                        instName = addr
```

Comment:
I jut double checked and the the USB scope's address already ends in '::INSTR'.

Is "KNOWN_DEVICE_TYPES" a global?  If so, I think I made these changes correctly in the next commit.